libs/shared-ui/src/lib/teacher-overview/teacher-overview.tsx

```diff
@@ -0,0 +1,68 @@
+import Avatar, { AvatarSize } from '../avatar/avatar';
+import Icon from '../icon/icon';
+import styles from './teacher-overview.module.scss';
+
+/* eslint-disable-next-line */
+export interface TeacherOverviewProps {
+  name: string;
+  image: string;
+  profession: string;
+  rating: number;
+  reviews: number;
+  students: number;
+  courses: number;
+  bibliography: string;
+}
+
+export function TeacherOverview(props: TeacherOverviewProps) {
+  const {
+    name,
+    profession,
+    rating,
+    reviews,
+    students,
+    courses,
+    image,
+    bibliography
```

Comment:
we have a typo it should be: biography
please check whole app in order to fix in other places